httpserver-conf.lua

```diff
@@ -13,7 +13,7 @@ conf.general.port = 80
 -- WiFi configuration
 conf.wifi = {}
 -- Can be wifi.STATION, wifi.SOFTAP, or wifi.STATIONAP
-conf.wifi.mode = wifi.STATION
+conf.wifi.mode = wifi.SOFTAP
```

Comment:
this change seems unrelated, can you revert? (or if related please explain)

Reply:
yup, it's unrelated. Just is a working default, as opposed to the station config that will result in a reboot loop, unless you have an open wifi named "Internet" ;-)
If you prefer a separate pull request, that's fine with me.